Makefile

```diff
@@ -1,9 +1,10 @@
-ETHEREUM_IBC_PROTO ?= ./ethereum-ibc-rs/proto/definitions
+DOCKER := $(shell which docker)
+
+protoVer=0.13.1
+protoImageName=ghcr.io/cosmos/proto-builder:$(protoVer)
+protoImage=$(DOCKER) run --user 0 --rm -v $(CURDIR):/workspace --workdir /workspace $(protoImageName)
 
 .PHONY: proto-gen
 proto-gen:
 	@echo "Generating Protobuf files"
-	@mkdir -p ./proto/ibc/lightclients/ethereum/v1
```

Comment:
Any reason to remove these?
The proto definitions of light client need to be copied from `ethereum-ibc-rs`.

Reply:
@dongrie I think that if you follow this advice you can avoid adding `cp -r github.com/datachainlab/ethereum-lcp/* ./` in `scripts/protocgen.sh`.

Reply:
Revert mkdir code, and remove cp command from protocgen.sh
https://github.com/datachainlab/ethereum-ibc-relay-prover/pull/3/commits/7484802a5ca3f65de988f15af8808db21dce92be

Reply:
I misunderstand the point. You also need revert sed code.

Reply:
Ah, but maybe, you also need change `go_package` from ethereum-lcp to ethereum-ibc-relay-prover.

Reply:
Revert sed
https://github.com/datachainlab/ethereum-ibc-relay-prover/pull/3/commits/4361f6b7b6141dbf088cc162572a7b833e66118f